### DISCUSSION TITLE
Dns Migrations Issues

### DISCUSSION THREAD

**Discussion by horlamiedea** [vercel/community#2878] [Help]
### Summary

A dns was connected to a Vercel project and then the project was later deleted. I created another project and hosted it and I tried to connect the same dns to the new project and I got this message "Another Vercel account is using this domain. Please set the following TXT record on _vercel.aquilasec.io to use aquilasec.io in this project. If you wish to [transfer](https://vercel.com/guides/how-can-i-move-a-domain-to-a-team#move-button) the entire domain, this will need to be done by the current domain owner. Once the verification is completed and the domain is successfully configured, the TXT record can be removed.". 
<img width="902" alt="Screenshot 2023-06-15 at 09 18 31" src="https://github.com/vercel/community/assets/47615821/05ed61d7-be53-44d0-a241-c1f00c00212b">

Please how can I rectify, I can't move the dns from the old project because it has been deleted

### Steps to Reproduce

A dns was connected to a Vercel project and then the project was later deleted. I created another project and hosted it and I tried to connect the same dns to the new project and I got this message "Another Vercel account is using this domain. Please set the following TXT record on _vercel.aquilasec.io to use aquilasec.io in this project. If you wish to [transfer](https://vercel.com/guides/how-can-i-move-a-domain-to-a-team#move-button) the entire domain, this will need to be done by the current domain owner. Once the verification is completed and the domain is successfully configured, the TXT record can be removed.". 

Please how can I rectify, I can't move the dns from the old project because it has been deleted

**Comment by smaeda-ks** [comment]
@horlamiedea Domain is back in your account now: https://vercel.com/dashboard/aquilaeth/domains
